frontend/views/components/GroupSettings/InviteActionButton.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+menu-parent
+  menu-trigger.is-icon
+    i.icon-ellipsis-v
+
+  menu-content.c-content
+    ul
+      menu-item(
+        tag='router-link'
+        item-id='hash-3'
+        to='/somewhere'
+      )
+        i18n.icon-vote-yea.c-icon(tag='i') See original proposal
```

Comment:
The way this i18n is used is not very correct because all text is inside `i` tag...
menu-item accepts an "icon" prop, so we can pass the icon name and everything is aligned automatically. You can see an example at `GroupMembers.vue`

Reply:
Fixed! :)